diff --git a/addon/routes/error.js b/addon/routes/error.js
index 077af09045ae8d59008b10ce53e765561c15dfe7..8156cbcd7dbced7a09eabfddb4381e05a23fd57f 100644
--- a/addon/routes/error.js
+++ b/addon/routes/error.js
@@ -2,4 +2,8 @@ import Route from '@ember/routing/route';

 export default class ErrorRoute extends Route {
   classNames = ['x404'];
+  setupController(x, error) {
+    console.log(`ended up on ErrorRoute`, error.message, error.stack);
+    super.setupController(...arguments);
+  }
 }
